### DISCUSSION TITLE
Multiple log files per application

### DISCUSSION THREAD

**Discussion by 8vp** [COVESA/dlt-daemon#674] [Q&A] [answered]
Hi all, 

Is there a mechanism within DLT Daemon where it can be configured to write DLT logs into different pre-defined files based on the application that logs. E.g. App1 commits log, all these logs will be written into App1.dlt, App2 commits log, all these logs will be written into App2.dlt. 

We are unable to see such a configuration in the dlt.cfg. Please advise/guide.

**Comment by minminlittleshrimp** [comment, marked as answer]
Hello @8vp 
I believe DLT already support that in logstorage filter.
By turning on logstorage and cofig filters for each app you can get the expected log files
Regards

**Reply by 8vp** [reply]
That is great thanks for answering! We missed to understand what that was, but that is clear now! Many thanks.
